Scripts/Swift/SetUpTestFirebase.swift

```diff
@@ -0,0 +1,16 @@
+let ConstantTeamId = "<#teamId#>"
+let ConstantStagingFirebaseAppId = "<#stagingFirebaseAppId#>"
+let ConstantFirebaseTesterGroups = "<#group1#>, <#group2#>"
+let ConstantMatchRepo = "git@github.com:{organization}/{repo}.git"
+
+let envMatchRepo = ProcessInfo.processInfo.environment["MATCH_REPO"] ?? ""
+let envStagingFirebaseAppId = ProcessInfo.processInfo.environment["STAGING_FIREBASE_APP_ID"] ?? ""
+let envTeamId = ProcessInfo.processInfo.environment["TEAM_ID"] ?? ""
+let firebaseTesterGroup = "nimble"
+
+let fileManager = FileManager.default
+
+try fileManager.replaceAllOccurrences(of: ConstantTeamId, to: envTeamId)
```

Comment:
if you agree with the above suggestions, we can omit `try` and will see error log if any